src/chainbridgeConfig.ts

```diff
@@ -50,78 +50,35 @@ export type ChainbridgeConfig = {
 export const chainbridgeConfig: ChainbridgeConfig = {
   // Local GETH <> Local Substrate
   chains: [
-    // {
-    //   chainId: 0,
-    //   networkId: 5,
-    //   name: "Ethereum - Local",
-    //   decimals: 18,
-    //   bridgeAddress: "0x62877dDCd49aD22f5eDfc6ac108e9a4b5D2bD88B",
-    //   erc20HandlerAddress: "0x3167776db165D8eA0f51790CA2bbf44Db5105ADF",
-    //   rpcUrl: "http://localhost:8545",
-    //   type: "Ethereum",
-    //   nativeTokenSymbol: "ETH",
-    //   tokens: [
-    //     {
-    //       address: "0x21605f71845f372A9ed84253d2D024B7B10999f4",
-    //       name: "TOKEN",
-    //       symbol: "TOKEN",
-    //       imageUri: ETHIcon,
-    //       resourceId:
-    //         "0x000000000000000000000000000000c76ebe4a02bbc34786d860b355f5a5ce00",
-    //     },
-    //   ],
-    // },
-    // {
-    //   chainId: 1,
-    //   networkId: 2,
-    //   name: "Substrate - Local",
-    //   decimals: 18,
-    //   rpcUrl: "ws://localhost:9944",
-    //   type: "Substrate",
-    //   nativeTokenSymbol: "DOT",
-    //   chainbridgePalletName: "chainBridge",
-    //   bridgeFeeFunctionName: "tokenTransferFee",
-    //   transferPalletName: "example",
-    //   transferFunctionName: "transferNative",
-    //   typesFileName: "bridgeTypes.json",
-    //   tokens: [
-    //     {
-    //       address: "substrate-native",
-    //       name: "DOT",
-    //       symbol: "DOT",
-    //       resourceId: "substrate-native",
-    //     },
-    //   ],
-    // },
     {
-      chainId: 0,
-      networkId: 42,
+      chainId: 5,
+      networkId: 5,
       name: "Ethereum",
       decimals: 18,
```

Comment:
Is it native token decimals? If no, update with 10

Reply:
Moved config changes to seperate PR